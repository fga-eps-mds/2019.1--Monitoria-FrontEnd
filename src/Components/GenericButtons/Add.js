import React from 'react';
import PropTypes from 'prop-types';
import { withStyles } from '@material-ui/core/styles';
import Fab from '@material-ui/core/Fab';
import AddIcon from '@material-ui/icons/Add';
import '../Feed/feed.css';
import RegisterMonitoring from '../RegisterMonitoring/RegisterMonitoring'

const styles = theme =>({
  
  fab: {
    margin: 0,
    top: 'auto',
    right: 20,
    bottom: 65,
    left: 'auto',
    position: 'fixed',
    margin: theme.spacing.unit,
  },
  
});

function ButtonSizes(props) {
  const { classes } = props;
  return (
      <Fab size="medium" style={{backgroundColor: '#1DA1F2',
          color: 'white'}} aria-label="Add" className={classes.fab}>
          
          <AddIcon />
      </Fab>
  );
}

ButtonSizes.propTypes = {
  classes: PropTypes.object.isRequired,
};

export default withStyles(styles)(ButtonSizes);