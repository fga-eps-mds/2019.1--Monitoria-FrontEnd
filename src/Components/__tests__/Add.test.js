import React from 'react';
import { shallow } from 'enzyme';
import Add from '../GenericButtons/Add';

describe('Testing Add component', () => {
    it('Test if Add renders correctly', () =>{
        const tree = shallow(<Add/>);
        expect(tree).toMatchSnapshot();
    });
});