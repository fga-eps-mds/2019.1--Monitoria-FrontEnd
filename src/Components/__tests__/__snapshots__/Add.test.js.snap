// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing Add component Test if Add renders correctly 1`] = `
<ButtonSizes
  classes={
    Object {
      "fab": "ButtonSizes-fab-1",
    }
  }
/>
`;
